client/post-editor/editor-drawer/index.jsx

```diff
@@ -308,8 +308,11 @@ const EditorDrawer = React.createClass( {
 	renderStatus() {
 		// TODO: REDUX - remove this logic and prop for EditPostStatus when date is moved to redux
 		const postDate = this.props.post && this.props.post.date
-				? this.props.post.date
-				: null;
+			? this.props.post.date
+			: null;
+		const postStatus = this.props.post && this.props.post.status
```

Comment:
Minor refactoring point, this could be cleaner with `_.get`:

```
const postDate = get( this.props.post, 'date', null );
const postStatus = get( this.props.post, 'status', null );
```

Not clear whether `null` is necessary fallback.